code/player/Player.cs

```diff
@@ -80,6 +80,11 @@ public override void Respawn()
             Role = new NoneRole();
             Credits = 0;
 
+            using(Prediction.Off())
+            {
+                OnPlayerSpawned(To.Single(this));
+            }
+
```

Comment:
So which things do we want to move to `OnPlayerSpawned`? Same as `OnPlayedDied`?

Reply:
What do you mean exactly? :) Currently, I can't think about stuff that has to be moved as it's just a mechanic to call code on the client-side

Reply:
Do we want to prefix them with `Client` then?

`public void OnPlayerSpawned()` -> `public void ClientOnPlayerSpawned()`